srcpkgs/budgie-desktop-view/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'budgie-desktop-view'
+pkgname=budgie-desktop-view
+version=1.2
+revision=1
+build_style=meson
+hostmakedepends="pkg-config intltool vala glib-devel"
+makedepends="libglib-devel gtk+3-devel vala-devel"
+short_desc="Budgie desktop icons application"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="Apache-2.0"
+homepage="https://github.com/BuddiesOfBudgie/budgie-desktop-view"
+distfiles="${homepage}/releases/download/v${version}/budgie-desktop-view-v${version}.tar.xz"
```

Comment:
Let's not use $homepage for new packages.

Reply:
Ok, I did it to shorten the line a bit and because it was already done in `budgie-desktop` to keep it consistent.
btw this is the first time that it's actually useful because homepage changed but distfiles did not :D

Reply:
I changed it now for all